src/hooks/web3/index.tsx

```diff
@@ -107,13 +109,11 @@ export function Provider({
   )
 }
 
-const onError = (error: Error) => console.error(error)
-
 function initializeWeb3ReactConnector<T extends Connector, P extends object>(
   Constructor: { new (options: P): T },
-  options?: Omit<P, 'actions'>
+  options: Omit<P, 'actions'>
 ): Web3ReactConnector<T> {
-  const [connector, hooks] = initializeConnector((actions) => new Constructor({ actions, onError, ...options } as P))
+  const [connector, hooks] = initializeConnector((actions) => new Constructor({ actions, ...options } as P))
```

Comment:
You should continue to explicitly pass `onError` to the other connectors, eg, on L135:

    return initializeWeb3ReactConnector(EIP1193, { provider, onError: console.error })

Reply:
good catch thanks